azafea/event_processors/endless/country.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020 - Endless
```

Comment:
```suggestion
# Copyright (c) 2020 - Endless OS LLC
```

(and in all other new copyright headers)

Reply:
OK!